docs/configuration/selecting-excluding.rst

```diff
@@ -84,3 +88,26 @@ Examples:
             exclude=["node_name+"],  # node_name and its children
         )
     )
+
+Using ``selector``
+--------------------------------
+
+The ``selector`` parameter is a string that references a `dbt YAML selector <https://docs.getdbt.com/reference/node-selection/yaml-selectors>`_ already defined in a dbt project.
+
+.. note::
+
```

Comment:
I think it is worth moving the `:note` to the top, just before the `Using selector` heading - and to also state that this is available from 1.3 onwards.

Reply:
I moved the note block up so it now looks like this: 
![image](https://github.com/astronomer/astronomer-cosmos/assets/79104794/8b39fc55-d9b3-46ff-b35f-3686b4f97c52)